zhaquirks/tuya/ts0601_valve.py

```diff
@@ -568,3 +569,74 @@ class GiexIrrigationStatus(t.enum8):
     .skip_configuration()
     .add_to_registry()
 )
+
+
+class RoyalGardineerWeatherDelay(t.enum8):
+    """Royal Gardineer Irrigation Valve weather delay enum."""
+
+    Disabled = 0x00
+    Delayed_24h = 0x01
+    Delayed_48h = 0x02
+    Delayed_72h = 0x03
+
+
+class RoyalGardineerTimerState(t.enum8):
+    """Royal Gardineer Irrigation Valve timer state enum."""
+
+    Disabled = 0x00
+    Active = 0x01
+    Enabled = 0x02
+
+
+(
+    TuyaQuirkBuilder("_TZE200_2wg5qrjy", "TS0601")
+    .tuya_onoff(dp_id=1)
+    # Should be TuyaPowerConfigurationCluster2AA, but it is broken at this time.
+    .tuya_battery(dp_id=7, power_cfg=TuyaPowerConfigurationCluster2AAA)
```

Comment:
We should be able to use `TuyaPowerConfigurationCluster2AA` now (fixed in https://github.com/zigpy/zha-device-handlers/pull/3554).
```suggestion
    .tuya_battery(dp_id=7, power_cfg=TuyaPowerConfigurationCluster2AA)
```